ui/app/src/modules/Preview/QuickCreate.tsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```

Comment:
Use the new copyright notice.

Don't resolve conversations that you didn't start, let the person that started it resolve it so they can look for changes more easily.

Reply:
Got it. Copyright updated.